src/_scss/pages/homepage/features/_covidFeature.scss

```
@@ -0,0 +1,238 @@
+
+            .feature-covid-official-spending-data__image-mobile {
+                display: block;
+                max-width: 75%;
+                margin: rem(20) rem(0) rem(20) rem(0);
+    
+                @include media ($medium-screen) {
+                    display: none;
+                }
+            }
+
+            .feature-covid-official-spending-data__image {
+                display: none;
+    
+                @include media ($medium-screen) {
+                    display: block;
+                    width: 35vw;
+                    height: 25vw;
+                    max-width: none;
+                }
+            }
+        }
+    }
+
+    //ROW 2: ADVANCED SEARCH FILTER + SPENDING PROFILE 
+    .advanced-search-and-spending-profile__wrapper {
+        display: block;
+        margin-top: rem(60);
+
+        @include media ($medium-screen) {
+            margin-top: rem(10);
+            margin-left: rem(103);
+            @include display(flex);
+            @include justify-content(center);
+
+        }
+
+        .advanced-search__content-wrapper {
+            margin-top: rem(100);
+
+            @include media ($medium-screen) {
+                max-width: 45%;
+                @include display(flex);
+                @include flex-direction(column);
+            }
+        }
+
+        .advanced-search__content-wrapper-right {
+            margin-top: rem(150);
+            margin-bottom: rem(100);
+
+            @include media ($medium-screen) {
+                max-width: 45%;
+                margin-top: rem(100);
+                margin-left: rem(30);
+                margin-bottom: rem(0);
+
+                display: flex;
+                flex-direction: column;
+            }
+        }
+
+        .feature-covid-item__image-wrapper {
+            @include display(flex);
+            @include justify-content(flex-start);
+
+            .feature-covid-item__image-mobile {
+                display: block;
+                max-width: 75%;
+                border: rem(1) solid #D6D7D9;
+                padding: rem(10);
+                background-color: #FFFFFF;
+                margin-top: rem(20);
+                margin-left: auto;
+                margin-right: auto;
+
+                @include media($medium-screen) {
+                    display: none;
+                }
+            }
+            .feature-covid-item__image {
+                display: none;
+
+                @include media($medium-screen) {
+                    display: block;
+
+                    max-width: 75%;
+                    border: rem(1) solid #D6D7D9;
+                    padding: rem(10);
+                    margin-top: rem(0);
+                    margin-bottom: 5%;
+                    background-color: #FFFFFF;
+                }
+            }
+        }
+
+        .adv-search-spending-profile__text {
+            @include media ($medium-screen) {
+                max-width: 85%;
```

Max Width is not gonna work in IE. Lets must make this width

```suggestion
                width: 85%;
```